pom.xml

```diff
@@ -12,7 +12,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <dropwizard.version>0.7.1</dropwizard.version>
-        <dropwizard.cassandra.version>4.0.0</dropwizard.cassandra.version>
+        <dropwizard.cassandra.version>3.0.2</dropwizard.cassandra.version>
```

Comment:
why the downgrade?

also the commit message is a bit odd, and the most significant part of the commit is on the second line instead of the first.